ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx

```diff
@@ -0,0 +1,38 @@
+import './styles.scss';
+
+const GOLDEN_GLOBAL = [1, 0.9999, 0.9888, 1, 1, 0.8888];
```

Comment:
i understand this is our mock data but a component will/should not be aware of data that doesn't realte to it. let move mock data out of this one and pass only hat is needed

Reply:
Moved into a mock object.